fixtures/redirects_redirect.json

```diff
@@ -0,0 +1,9 @@
+[
+{
+  "model": "redirects.redirect",
+  "fields": {
+    "old_path": "/sk/pripadove-studie/",
+    "new_path": "/sk/naco-infozakon/"
```

Comment:
@viliambalaz Pls sprav ten json ako dict zobrazujuci stare URL na nove.

Ked su redirecty v povodnom formate, ze iba zoznam objektov, tak sa v tych redirectoch bude zle vyznat, ked ich bude vela. Bude v nich chaos. Kedze redirecty su vlastne zobraze z URL na URL, tak je ovela prehladnejsie, ked su definovane ako dict s usporiadanymi redirectovanymi URL. Ovela jednoduchsie sa potom tie redirecty doplnaju a udrzuje v nich poriadok.

Sprav to teda rovnako ako na tom druhom projekte.

Reply:
:ok: sorry, najprv som tomu zle porozumel